vignettes/customization.Rmd

```diff
@@ -92,6 +92,8 @@ Rather than picking an entire theme, you can customise individual theme componen
 template:
   bootstrap: 4
   bslib:
+    font_scale: 1.2
+    font_link: !expr bslib::font_google("muli")
```

Comment:
Should use new new syntax that doesn't need `!expr`: https://github.com/rstudio/bslib/issues/256

Reply:
Also it should be `font_base` not `font_link` (I think?)

Reply:
Thanks both, I updated the syntax following https://rstudio.github.io/bslib/articles/bslib.html#main-colors-fonts